app/assets/javascripts/discourse/app/controllers/topic.js

```diff
@@ -215,15 +215,24 @@ export default Controller.extend(bufferedProperty("model"), {
     if (posts) {
       posts
         .filter(
-          (p) =>
-            p.bookmarked &&
-            p.bookmark_auto_delete_preference ===
+          (post) =>
+            post.bookmarked &&
+            post.bookmark_auto_delete_preference ===
               AUTO_DELETE_PREFERENCES.ON_OWNER_REPLY
         )
-        .forEach((p) => {
-          p.clearBookmark();
+        .forEach((post) => {
+          const bookmarkId = post.bookmark_id;
```

Comment:
Trivial but there isn't much point in making a new variable to only use it once.